npm/design-system/src/components/FileExplorer/FileExplorerItem.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { ReactNode } from 'react'
+import { InlineIcon } from '@iconify/react'
+import javascriptIcon from '@iconify/icons-vscode-icons/file-type-js-official'
+import typescriptIcon from '@iconify/icons-vscode-icons/file-type-typescript-official'
+import reactJs from '@iconify/icons-vscode-icons/file-type-reactjs'
+import reactTs from '@iconify/icons-vscode-icons/file-type-reactts'
+import folderClosed from '@iconify/icons-vscode-icons/default-folder'
+import folderOpen from '@iconify/icons-vscode-icons/default-folder-opened'
+import { FolderOrFile } from './types'
+import styles from './FileExplorer.module.scss'
+import cs from 'classnames'
+
+const icons: Record<string, any> = {
```

Comment:
```suggestion
const icons: Record<string, { icon: typeof reactTs }> = {
```

Or pass the right type

Reply:
Thanks. There's a lot I don't really understand how to do.